modules/backend/src/main/java/com/lilium/springsecurityangular/user/UserRepository.java

```diff
@@ -5,6 +5,6 @@
 
 @Repository
 public interface UserRepository extends JpaRepository<UserEntity, Long> {
-
+    /*How about Optional<UserEntity> here for further use elsewhere? */
```

Comment:
Sure that can be done, but I do not see much benefit of it as you need to do null checks anyway. Since this is aimed at beginners I think leaving optional out of it makes it easier to understand